.github/workflows/apt-packages.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        image: [ "debian:10-slim", "debian:11-slim", "debian:12-slim", "ubuntu:20.04", "ubuntu:22.04", "ubuntu:23.04" ]
+        image: [ "debian:10-slim", "debian:11-slim", "debian:12-slim", "ubuntu:20.04", "ubuntu:22.04", "ubuntu:24.04" ]
```

Comment:
should we drop 20.04 while we are at it?

Reply:
we still support 20.04 so would be good to keep it running in the main branch in order to make the release captain life easier when working on packaging to avoid and last time extra work dealing with potential issues.

Reply:
20.04 EOL is in 2032 so I doubt we should drop it

Reply:
https://ubuntu.com/about/release-cycle